examples/microprofile/mp1_1-static-content/src/main/java/io/helidon/microprofile/example/staticc/Main.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2020 Oracle and/or its affiliates.
```

Comment:
This main class seems to be useless.

Reply:
Also type in package name

Reply:
`staticc`? I think that is short for static content. I will leave this as-is for now.